docs/docsite/rst/roadmap/COLLECTIONS_8.rst

```diff
@@ -40,6 +40,12 @@ Release schedule
 
   Breaking changes will be introduced in Ansible 8.0.0, although we encourage the use of deprecation periods that will show up in at least one Ansible release before the breaking change happens, this is not guaranteed.
 
+.. note::
+
+  When a prerelease of Ansible 8 happens in the same week as an Ansible 7 release, the Ansible 8 prerelease will happen on the day after the Ansible 7 release.
+  When a release of Ansible 8 happens in the same week as an Ansible 9 release, the Ansible 8 release will happen on the day after the Ansible 9 release.
+  In general it's in the discretion of the release manager to delay a release by 1-2 days for reasons such as personal (schedule) problems, technical problems (CI/infrastructure breakdown), and so on.
```

Comment:
@mariolenz I reformatted and slightly rewrote your suggestion:
```suggestion
  In general it's in the discretion of the release manager to delay a release by 1-2 days for reasons such as personal (schedule) problems, technical problems (CI/infrastructure breakdown), and so on.
  However, in case two releases are planned for the same day, a release of the latest stable version takes precedence. This means that if a stable Ansible 8 release collides with a pre-release of Ansible 9, the latter will be delayed.
  If a Ansible 8 release collides with a stable Ansible 9 release, including 9.0.0, the Ansible 8 release will be delayed.
```
What do you think?

Reply:
LGTM!

@anweshadas @cyberark-bizdev What do you think?

Reply:
LGTM, thank you both!